### ISSUE TITLE
Emit a generic_error bad row when an uncaught exception is sent to Sentry

### ISSUE THREAD

**Issue by benjben** [snowplow/enrich#344]
At the moment no bad row is emitted when an uncaught exception is thrown by Common Enrich (e.g. `NullPointerException`).

**Comment by chuwy** [comment]
Good idea. Doing it in Stream Enrich NH already: https://github.com/snowplow/enrich/pull/328/commits/b1b6d794546b41876157400308a59ba78313f3d5#diff-95f76083a48d52e708fa4ebcfc54762fR91-R110
